src/components/ecommerce/movieDetails/movieScreenshotListItem.component.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {
+  Image,
+  ImageProps,
+} from 'react-native';
+import {
+  ThemedComponentProps,
+  ThemeType,
+  withStyles,
+} from '@kitten/theme';
+
+export type MovieScreenshotListItemProps = ThemedComponentProps & ImageProps;
+
+class MovieScreenshotListItemComponent extends React.Component<MovieScreenshotListItemProps> {
+
+  public render(): React.ReactNode {
```

Comment:
Does it make sense to move it to the separate component?

Reply:
Not really. Agree with that
But it's better to do so while we have more complex `ListItem`s which are separated from `List`s